ih8sn.conf.kebab

```diff
@@ -0,0 +1,2 @@
+BUILD_FINGERPRINT=OnePlus/OnePlus8T/OnePlus8T:13/RKQ1.211119.001/R.ed8c2c_1_2:user/release-keys
```

Comment:
remove this file

Reply:
Why remove, it should be updated?

Reply:
Updated.

https://github.com/althafvly/ih8sn/pull/6/commits/43d5062a9eaea8dc602e83ee5aa21add79654c57

Reply:
you are commiting two files. remove one